src/encoded/static/components/item.js

```diff
@@ -19,11 +22,15 @@ const Fallback = (props, reactContext) => {
                 </div>
             </header>
             {typeof context.description === 'string' ? <p className="description">{context.description}</p> : null}
-            <section className="view-detail panel">
-                <div className="container">
-                    <pre>{JSON.stringify(context, null, 4)}</pre>
-                </div>
-            </section>
+            {path === '/profiles/' ?
```

Comment:
Is this the preferred way to break a conditional?

Reply:
If you mean the line break, it’s the way we established quite a while ago because it seemed most easy formatting-wise when the true condition is long. AirBnB doesn’t have anything to say about it. React doesn’t recommend anything, but their examples are similar but they add some parentheses. I _think_ older React documentation didn’t use parens.

https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator